### PR TITLE
feat: handle image drop on editor

### PR DESCRIPTION
**Pull request by gregogun**
- Adds logic to handle drag and drop functionality on the editor
- Handles multiple images being dropped on the editor
- Adds improved loading state via a node view wrapper around image element

Fixes https://github.com/sigle/sigle/issues/652

### CODE REVIEW

**Review by pradel — COMMENTED** [review]
Code lgtm ✅ 

Love the new loading component, the drag and drop experience is really smooth ❤️.

But I feel like now, it's much harder to see what's going on when you drag an existing image in the editor and move it to some block below. As the drop line color is really close to the background I couldn't figure in which block my image was about to be dropped. While comparing to the current version, I prefer a lighter color so you can really see it with the dark background. Wdyt?

**Comment by gregogun** [comment]
> Code lgtm ✅
> 
> Love the new loading component, the drag and drop experience is really smooth ❤️.
> 
> But I feel like now, it's much harder to see what's going on when you drag an existing image in the editor and move it to some block below. As the drop line color is really close to the background I couldn't figure in which block my image was about to be dropped. While comparing to the current version, I prefer a lighter color so you can really see it with the dark background. Wdyt?

Spoke to Quentin about this one and increased it by a step (now at step 8), wdyt? If it's still not enough can see with him about pushing it to 9.

**Review by pradel — APPROVED** [review]
LGTM, once the conflicts are resolved let's merge 🚀

**Pull request merged**
